test/integration/targets/openssl_certificate/tasks/main.yml

```diff
@@ -7,7 +7,8 @@
       openssl_csr:
         path: '{{ output_dir }}/csr.csr'
         privatekey_path: '{{ output_dir }}/privatekey.pem'
-        commonName: 'www.ansible.com'
```

Comment:
Could you leave one of the test with the previous interface, so we know it still does work. I've tried it locally and I know it works but so that test also shows (and ensure) that.